!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  ece_max_size: 5000.0
  ece_min_size: 200.0
  ece_tolerance: 0.02
  groups: !!python/object/new:dynamic_reconfigure.encoding.Config
    dictitems:
      groups: !!python/object/new:dynamic_reconfigure.encoding.Config
        dictitems:
          EuclideanClusterExtraction: !!python/object/new:dynamic_reconfigure.encoding.Config
            dictitems:
              ece_max_size: 5000.0
              ece_min_size: 200.0
              ece_tolerance: 0.02
              groups: !!python/object/new:dynamic_reconfigure.encoding.Config
                state: []
              id: 3
              name: EuclideanClusterExtraction
              parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
                state: []
              parent: 0
              state: true
              type: ''
            state: []
          PassThrough: !!python/object/new:dynamic_reconfigure.encoding.Config
            dictitems:
              groups: !!python/object/new:dynamic_reconfigure.encoding.Config
                state: []
              id: 1
              name: PassThrough
              parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
                state: []
              parent: 0
              pt_max_x: 0.2
              pt_max_y: 0.0
              pt_max_z: 0.0
              pt_min_x: -0.2
              pt_min_y: 0.0
              pt_min_z: 0.0
              pt_x: true
              pt_y: false
              pt_z: false
              state: true
              type: ''
            state: []
          SacSegmentation: !!python/object/new:dynamic_reconfigure.encoding.Config
            dictitems:
              groups: !!python/object/new:dynamic_reconfigure.encoding.Config
                state: []
              id: 2
              name: SacSegmentation
              parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
                state: []
              parent: 0
              sacs_distance_threshold: 0.01
              sacs_method: 0
              sacs_model: 0
              state: true
              type: ''
            state: []
          SvmClassification: !!python/object/new:dynamic_reconfigure.encoding.Config
            dictitems:
              groups: !!python/object/new:dynamic_reconfigure.encoding.Config
                state: []
              id: 5
              name: SvmClassification
              parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
                state: []
              parent: 0
              state: true
              svmc_path_to_model: /home/lstern/dhbw-screwing-robot/intelrealsense_d435/data/0_grad/recording_1/svm_model.txt
              svmc_training: true
              type: ''
            state: []
          VfhEstimation: !!python/object/new:dynamic_reconfigure.encoding.Config
            dictitems:
              groups: !!python/object/new:dynamic_reconfigure.encoding.Config
                state: []
              id: 4
              name: VfhEstimation
              parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
                state: []
              parent: 0
              state: true
              type: ''
              vfhe_normal_radius_search: 0.05
            state: []
        state: []
      id: 0
      name: Default
      parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      parent: 0
      state: true
      type: ''
    state: []
  pt_max_x: 0.2
  pt_max_y: 0.0
  pt_max_z: 0.0
  pt_min_x: -0.2
  pt_min_y: 0.0
  pt_min_z: 0.0
  pt_x: true
  pt_y: false
  pt_z: false
  sacs_distance_threshold: 0.01
  sacs_method: 0
  sacs_model: 0
  svmc_path_to_model: /home/lstern/dhbw-screwing-robot/intelrealsense_d435/data/0_grad/recording_1/svm_model.txt
  svmc_training: true
  vfhe_normal_radius_search: 0.05
state: []
